tests/fixtures/squares.sql

```diff
@@ -6,7 +6,7 @@ CREATE OR REPLACE FUNCTION squares(
     ymax float,
     epsg integer,
     -- additional parameters
-    depth integer default 2
+    query_params json
```

Comment:
this is the same approach as in https://github.com/urbica/martin#function-sources

Basically, we forward the additional parameter (`kwargs`) to the SQL function and it's up the the user to know what to do with it 🤷‍♂️

Reply:
Yes, I like this pattern (it's basically what I am doing with pgstac)